openpype/settings/entities/op_version_entity.py

```diff
@@ -0,0 +1,85 @@
+from openpype.lib.openpype_version import (
+    get_remote_versions,
+    get_OpenPypeVersion,
+    get_build_version
+)
+from .input_entities import TextEntity
+from .lib import (
+    OverrideState,
+    NOT_SET
+)
+from .exceptions import BaseInvalidValue
+
+
+class OpenPypeVersionInput(TextEntity):
+    """Entity to store OpenPype version to use.
+
+    It is text input as creating of settings on different machines may
```

Comment:
```suggestion
   Settings created on another machine may affect available versions on an current user's machine.
    Text input element is provided to explicitly set version not yet showing up the an user's machine.
```
Or something like that.